### ISSUE TITLE
Distinguish C# and F# code in documentation by background colour

### ISSUE THREAD

**Issue by hickford** [fscheck/FsCheck#439]
An idea: it could be helpful to distinguish C# code from F# code in the documentation by giving it a different background colour. This would make the documentation easier to read. I know both F# and C# but still find it confusing reading one when expecting another.

Example https://fscheck.github.io/FsCheck/Properties.html

**Comment by kurtschelfthout** [comment]
Yes, this is a good idea. Do you know how to do it? We use Fsharp.formatting. Perhaps there is some css magic we can do?

**Comment by jzabroski** [comment]
Just add a tab on top that says f# or c#

**Comment by Angr1st** [comment]
I would like to tackle this.

**Comment by kurtschelfthout** [comment]
Nice! Do you need some pointers - I can only be of minimal help - I would start with digging through the generated html etc and figure out if there is some css we can leverage. 

FSharp.formatting does allow us to specify a template so maybe it’s possible to put something there.

**Comment by Angr1st** [comment]
I have already skimmed through the code and tried to find out how to alter the css to adapt the background color. It currently seems like we just use the default css that is delivered by Fsharp.Formatting. I have opened an issue with them for this as a feature as I think that there would be more people that would enjoy some clearer language snippet differentiation.

**Comment by kurtschelfthout** [comment]
That's awesome.

I noticed that the snippets are surrounded by ```<code lang="fsharp"></code>``` tags - any chance we can do something with that?

Sorry if that's stupid or obvious, I know next to nothing about css etc :)

**Comment by Angr1st** [comment]
No worries my css knowledge is not that big but in theory we could but it would need to be done in fsharp.formatting itself.
